integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/K8sEvents.java

```diff
@@ -217,6 +218,37 @@ public static CoreV1Event getEvent(String opNamespace, String domainNamespace, S
     return null;
   }
 
+  /**
+   * Get matching event object with specific reason and type.
+   * @param domainNamespace namespace in which the event is logged
+   * @param reason event to check for Created, Changed, deleted, processing etc
+   * @param type type of event, Normal or Warning
+   * @param timestamp the timestamp after which to see events
+   * @return CoreV1Event matching event object
+   */
+  public static List<CoreV1Event> getEvents(String domainNamespace,
+                                            String reason,
+                                            String type,
+                                            OffsetDateTime timestamp) {
+
+    List<CoreV1Event> events = new ArrayList<>();
+
+    try {
+      List<CoreV1Event> allEvents = Kubernetes.listNamespacedEvents(domainNamespace);
```

Comment:
Use listOpGeneratedNamespacedEvents to only get operator generated events.

Reply:
where can I can this listOpGeneratedNamespacedEvents method?

Reply:
Change listNamespacedEvents to listOpGeneratedNamespacedEvents. Both are utility methods in Kubernetes.java.

Reply:
Sorry, I just realized that this is for release/3.3, where the new method does not exist.

Reply:
added listOpGeneratedNamespacedEvents in Kubernetes.java